### ISSUE TITLE
Error after click "Embed Script Lab"

### ISSUE THREAD

**Issue by BurdenBear** [OfficeDev/Office-OOXML-EmbedAddin#2]
## Error Describtion:
I followed step to embed add-in in excel file, and after click "Embed Script Lab" button,
I encount the following error:
(I changed the code only to let it print detail exception tracestack)
```
ERROR: DocumentFormat.OpenXml.Packaging.OpenXmlPackageException: Only one instance of the type is allowed for this parent. 
在 DocumentFormat.OpenXml.Packaging.OpenXmlPartContainer.InitPart[T](T newPart, String contentType, String id) 
在 DocumentFormat.OpenXml.Packaging.OpenXmlPartContainer.InitPart[T](T newPart, String contentType) 
在 DocumentFormat.OpenXml.Packaging.SpreadsheetDocument.AddWebExTaskpanesPart() 
在 Office_OOXML_EmbedAddin.AddinEmbedder.EmbedAddin(String fileType, MemoryStream memoryStream, String snippetID) 
位置 C:\Users\Morri\Documents\NodeProjects\Office-OOXML-EmbedAddin\Office-OOXML-EmbedAddin\AddinEmbedder.cs:行号 22 
在 Office_OOXML_EmbedAddin.WebForm1.btnEmbed_Click(Object sender, EventArgs e) 
位置 C:\Users\Morri\Documents\NodeProjects\Office-OOXML-EmbedAddin\Office-OOXML-EmbedAddin\Home.aspx.cs:行号 86
```
I have tried some .xlsx files but get the same error

## Environment:
system: windows 10
Microsoft Excel: 2016 MSO (16.0.11901.20070)
visual studio 2019 

![image](https://user-images.githubusercontent.com/5682650/62420153-3757fc80-b6c0-11e9-9042-81f1c6ab3d49.png).

Could any one help me with this problem? Thanks very much!

**Comment by BurdenBear** [comment]
fix in #3

**Comment by Rick-Kirkham** [comment]
Change is merged into master.
